plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -204,23 +203,6 @@ protected Optional<DataMappingTestSetup> filterDataMappingSmokeTestData(DataMapp
         return Optional.of(dataMappingTestSetup);
     }
 
-    @Override
-    protected Optional<String> filterColumnNameTestData(String columnName)
```

Comment:
Do we have column name tests with `none` column mapping mode in product tests? I remember there's `TestDeltaLakeColumnMappingMode#testSpecialCharacterColumnNamesWithColumnMappingMode` for `id` and `name` mappings, but I'm not sure about existing `none` mapping tests. 

As you know, we create tables and verify the result by ourselves in BaseDeltaLakeMinioConnectorTest, so it doesn't guarantee whether the column is accessible from Databricks or OSS Delta Lake. 

Removing this override itself is fine.

Reply:
I cannot find any. It would be good follow up I guess.